app/src/main/java/gitlin/kothub/ProfileActivity.kt

```diff
@@ -28,10 +28,10 @@ class ProfileActivity : AppCompatActivity(), AnkoLogger {
                 username.text = summary.login
                 description.text = summary.bio
 
-                followers.text = summary.followers?.toString()
-                stars.text = summary.starredRepositories?.toString()
-                repos.text = summary.repositories?.toString()
-                following.text = summary.following?.toString()
+                followers.text = summary.toString(summary.followers)
```

Comment:
Would be nice to avoid writting toString altogether.

What I thought about: 

```
fun TextView.setText(value: Int) = setText(value.toString())
```

To see if it's possible to create an extension method that overload an existing one.